stripper/ze_ffxii_feywood_b6_3k.cfg

```diff
@@ -1,3 +1,316 @@
+	"origin" "9096 12305 5523.5"
+	"model" "*411"
+	"shadowdepthnocache" "0"
+	"damagetoenablemotion" "0"
+	"Damagetype" "0"
+	"ExploitableByPlayer" "0"
+	"disableflashlight" "0"
+	"disableshadowdepth" "0"
+	"disablereceiveshadows" "0"
+	"fademaxdist" "0"
+	"disableshadows" "0"
+	"drawinfastreflection" "0"
+	"ExplodeDamage" "0"
+	"explodemagnitude" "0"
+	"ExplodeRadius" "0"
+	"explosion" "0"
+	"fademindist" "-1"
+	"fadescale" "1"
+	"spawnobject" "0"
+	"forcetoenablemotion" "0"
+	"spawnflags" "34817"
+	"gibdir" "0 0 0"
+	"health" "0"
+	"rendermode" "10"
+	"massScale" "0"
+	"material" "0"
+	"renderfx" "0"
+	"nodamageforces" "0"
+	"rendercolor" "255 255 255"
+	"notsolid" "0"
+	"renderamt" "255"
+	"propdata" "0"
+	"PerformanceMode" "0"
+	"pressuredelay" "0"
+	"preferredcarryangles" "0 0 0"
+}
+
+;Move cactus free level to button instead of a trigger, as you can press the button and someone else steals the level (or multiple get free level cause all touching trigger_once)
+filter:
+{
+	"classname" "trigger_once"
+	"targetname" "Cactus_Trigger"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "Cactus_Button"
+	}
+	delete:
+	{
+		"OnPressed" "Cactus_TriggerEnable0-1"
+	}
+	insert:
+	{
+		"OnPressed" "!activatorAddOutputOnUser4 Map_Level_Check:Add:1:0.00:-10-1"
+		"OnPressed" "cmdCommandsay **Cactus found. The player who did it got 1 level**0-1"
+		"OnPressed" "Map_Score_AddApplyScore0-1"
+		"OnPressed" "Cactus_ButtonKill0-1"
+		"OnPressed" "Map_Local_GametextAddOutputmessage YOU FOUND THE CACTUS. YOU GOT 100 EXP AND LEVELED UP0-1"
+		"OnPressed" "Map_Local_GametextDisplay0.5-1"
+		"OnPressed" "Map_Local_GametextAddOutputcolor 49 255 130-1"
+		"OnPressed" "Map_Local_GametextAddOutputcolor 234 255 2302-1"
+		"OnPressed" "Cactus*Kill0.5-1"
+		"OnPressed" "Sidequest_CactusKill0.25-1"
+		"OnPressed" "Sidequest_Cactus_CaseKill0.25-1"
+	}
+}
+
+;Change the func_button for berserk mode toggling to a game_ui with right click (like other feywood ports/versions), so that entwatch doesnt track it
+filter:
+{
+	"classname" "func_button"
+	"targetname" "Item_Berserk_Change"
+}
+
+filter:
+{
+	"classname" "filter_activator_name"
+	"targetname" "Item_Berserk_Filter_2"
+}
+
+add:
+{
+	"classname" "game_ui"
+	"targetname" "Item_Berserk_UI"
+	"origin" "-1904 -6688 -4610"
+	"spawnflags" "0"
+	"FieldOfView" "-1.0"
+	"PressedAttack2" "Item_Berserk_Relay_gameui,Trigger,,0,-1"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "game_text"
+		"targetname" "Item_Berserk_Gametext"
+	}
+	replace:
+	{
+		"message" "You picked up Berserk Magick. Right click (+attack2) to switch modes (ammo/speed)"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "weapon_elite"
+		"targetname" "Weapon_Berserk"
+	}
+	insert:
+	{
+		"OnPlayerPickup" "Item_Berserk_UI,Activate,,0.05,-1"
+		"OnPlayerPickup" "player,FireUser2,,0,-1"
+		"OnPlayerPickup" "!activatorAddOutputOnUser2 Item_Berserk_UI:Deactivate::0:10-1"
```

Comment:
You should delay the OnUser2 AddOutput more than the FireUser2 for safety reasons, even just by 0.02 seconds. Else the output could be added and fired in the same frame, if it managed to executes in that order.